Add support for [Add|Remove]-QueryIndex commands

philrowan opened the pull request:
Added 2 functions
Add-QueryIndex
Remove-QueryIndex

I am not familiar with this PS test stuff. Can you point me to any docs so I can write tests please?

philrowan committed:
Add/Remove QueryIndex

Add functions for Adding and Removing Query Indexed

philrowan committed:
fix docs

matthewhatch commented:
They're written using Pester which is a testing framework written in powershell for powershell. https://github.com/pester/Pester

philrowan committed:
Add Tests for QueryIndex methods

Add tests for testing inputs to new Add-QueryIndex and Remove-QueryIndex
methods

philrowan commented:
I've added tests for the input parameters similar to the tests for the other functions and a test for  optional parameters with default values in the Add-QueryIndex function.
